handlers/retrieve_resource.go

```diff
@@ -32,13 +31,13 @@ func (d *resourceEntry) Handle(params operations.RetrieveResourceParams) middlew
 }
 
 // TODO: expand this mapping
-func buildResponse(resource *persistence.Resource) *models.Resource {
+func buildResponse(resource *models.Resource) *models.Resource {
```

Comment:
not for this PR, but i'm pretty sure we don't need a buildResponse function - just returning the retrieved data as a `interface{}` worked for me.

Reply:
Aaron and I had the same thoughts. A subsequent PR will probably end up removing this method.